tests/pos/rvec.rs

```diff
@@ -1,3 +1,5 @@
+#![feature(register_tool)]
```

Comment:
Do we really need this here? I think we only need to register the tool in the crate root.

Reply:
Good point I removed it.